.travis.yml

```diff
@@ -63,6 +68,10 @@ install:
 script:
   - ./scripts/ci/test.sh -o $HOST_OS -r $TARGET_ARCH
   - ./scripts/ci/build-installers.sh -o $HOST_OS -r $TARGET_ARCH
+  - if [[ ${TRAVIS_BRANCH} == "master" ]] && [[ ${TRAVIS_PULL_REQUEST} == "false" ]]; then
```

Comment:
Yeah, otherwise any user could put malicious Docker commands in the cache. See https://github.com/resin-io/etcher/pull/1695 for the Linux check fix, plus other nasty things I found while debugging this.